### PR TITLE
[WIP] Migrate to python3 🎉🐍

### PR DESCRIPTION
**Pull request by abulte**

### CODE REVIEW

File: setup.cfg [hunk 1]
```diff
@@ -27,9 +27,11 @@ norecursedirs = .git build .cache udata_gouvfr/theme udata_gouvfr/translations
 python_files = tests.py
 python_functions = test_*
 python_classes = *Test
+mock_use_standalone_module = true
 
 [flake8]
 exclude = doc,.git
+max-line-length = 100
```
Comment (noirbizarre) [inline setup.cfg]: :tada:

File: tasks.py [hunk 2]
```diff
@@ -98,7 +95,7 @@ def qa(ctx):
         else:
             success('No lint to fix')
         info('Ensure PyPI can render README and CHANGELOG')
-        readme_results = ctx.run('python setup.py check -m -s', pty=True, warn=True, hide=True)
+        readme_results = ctx.run('python3 setup.py check -m -s', pty=True, warn=True, hide=True)
```
Comment (noirbizarre) [inline tasks.py]: I believe that if the venv is created with/for python 3, there is no need to call `python3`, `python` will be aliased to the `python3` executable

File: tox.ini [hunk 3]
```diff
@@ -4,13 +4,13 @@
 # and then run "tox" from this directory.
 
 [tox]
-envlist = py27, doc
+envlist = py36, doc
 
 [testenv]
 commands =
-    python setup.py develop
+    python3 setup.py develop
```
Comment (noirbizarre) [inline tox.ini]: Same here: `python` and `pip` will be properly aliased to `python` and `pip3`

File: udata_gouvfr/harvesters/maaf.py [hunk 4]
```diff
@@ -111,7 +108,7 @@
 
 
 def extract(element):
-    lst = filter(lambda r: isinstance(r[0], basestring), map(dictize, element))
+    lst = filter(lambda r: isinstance(r[0], str), map(dictize, element))
```
Comment (noirbizarre) [inline udata_gouvfr/harvesters/maaf.py]: List comprehension ?

File: udata_gouvfr/tests.py [hunk 5]
```diff
@@ -445,7 +443,7 @@ def test_oembed_town_territory_api_get(self, api):
             assert cgi.escape(territory.url) in html
             assert 'alt="{name}"'.format(name=organization.name) in html
             assert md(territory.description, source_tooltip=True) in html
-            assert 'Download from localhost' in html
+            assert 'Download from local.test' in html
```
Comment (noirbizarre) [inline udata_gouvfr/tests.py]: Hum, those changes should be on the current master :/ (I will submit the PR unless you do)

File: tasks.py (hunk 2 above)
Comment (abulte) [reply tasks.py]: True, but I like the "expressiveness" of `python3`. YMMV.

File: udata_gouvfr/harvesters/maaf.py (hunk 4 above)
Comment (abulte) [reply udata_gouvfr/harvesters/maaf.py]: 👍

File: udata_gouvfr/tests.py (hunk 5 above)
Comment (abulte) [reply udata_gouvfr/tests.py]: Please do :-)

File: udata_gouvfr/harvesters/maaf.py (hunk 4 above)
Comment (abulte) [reply udata_gouvfr/harvesters/maaf.py]: PS: this is not tested :-( I hope I won't break anything.

File: tasks.py (hunk 2 above)
Comment (abulte) [reply tasks.py]: I’ve been thinking about this, I’ll revert to `python` everywhere, including udata core. 👌

**Pull request closed without merging**